server/src/pages/shot/view.js

```diff
@@ -250,11 +268,16 @@ class Body extends React.Component {
     let deleteTime = new Date(expireTime + this.props.retentionTime);
     let restoreWidget;
     if (this.props.isOwner) {
+      // todo l10n - timediffs
       restoreWidget = (
         <p>
-          If you do nothing,
-          this shot will be permanently deleted in <TimeDiff date={deleteTime} />.
-          <button className="button primary" onClick={this.onRestore.bind(this)}>restore for {intervalDescription(this.props.defaultExpiration)}</button>
+          <Localized id="shotPageExpirationMessage">
+            If you do nothing,
+            this shot will be permanently deleted in <TimeDiff date={deleteTime} />.
```

Comment:
Good catch. Because the timediff is a placeholder in the ftl file, I think it's best to explicitly make it a variable attribute.